common/src/main/java/com/wynntils/functions/HorseFunctions.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions;
+
+import com.wynntils.core.functions.ActiveFunction;
+import com.wynntils.mc.utils.InventoryUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import com.wynntils.wc.custom.item.properties.ItemProperty;
+import net.minecraft.world.item.ItemStack;
+
+public class HorseFunctions {
+
+    public static class horseLevelFunction extends ActiveFunction<String> {
```

Comment:
Please use conventional naming, i.e. UpperCamelCase, for classes, even nested classes.